src/test/java/uk/gov/pay/directdebit/common/clients/GoCardlessClientFacadeTest.java

```diff
@@ -100,4 +112,23 @@ public void validateReturnsGoCardlessBankAccountLookupWithNullBankNameAndBacsFal
         assertThat(result.isBacs(), is(false));
     }
 
+    @Test
+    public void createMandateReturnsGoCardlessMandateWithCreditorId() {
```

Comment:
Since this is the only test that tests the `GoCardlessClientWrapper.createMandate(MandateExternalId, GoCardlessCustomer)` method, we may as well want to make it a more general test and have it assert other stuff we know about the returned mandate (e.g. the IDs and reference).